src/signal.rs

```diff
@@ -55,6 +55,12 @@ use {
     WidgetHelpType,
 };
 
+#[cfg(gtk_3_16)]
+use {
+    gdk,
```

Comment:
Sure, actually removing the trait as mentioned in your other comment meant I could remove both of those.